heron/tools/ui/resources/templates/application.html

```diff
@@ -1,4 +1,4 @@
-<!--
+{% comment %}
```

Comment:
isn't it [`{# … #}`](https://jinja.palletsprojects.com/en/2.10.x/templates/#comments)?

Reply:
I found the syntax [here](https://blog.j95.io/3/). Apologies if it's the wrong format. I'll make the update.